### ISSUE TITLE
[BITV] 9.4.1.2/3.11 - The "Sharing" tab - "Actions" menu has been implemented as an application menu with the role="menu". However, the implementation is not complete as the role="menuitem" is missing. It is recommended to forgo the implementation as an application menu and remove the role="menu". (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37108]
- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v3-n11

</details>

- [ ] 9.4.1.2/10.11 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v10-n11

**Comment by JuliaKirschenheuter** [comment]
Hi @michaelnissenbaum 

I would like to ask you to test an "Actions" menu a bit because it is looks to me that this issue is not relevant anymore: 

![Screenshot from 2023-10-19 15-28-19](https://github.com/nextcloud/server/assets/6078378/715c5786-f8f8-4945-a508-db95ed6a84bf)

As far i can see there is `role="menuitem"` inside of main menu with `role="menu"`. Or have i missed something?

Thanks a lot!

**Comment by michaelnissenbaum** [comment]
Hi @JuliaKirschenheuter The output of the screen reader appears to be fine to me. The only issue I have found is that navigation with arrow keys does not work as expected - please see the attached screenshots with descriptions.

![CleanShot 2023-10-19 at 21 40 35@2x](https://github.com/nextcloud/server/assets/48948959/19cb4f9d-43d9-4f65-9664-bd5cdd475661)

![CleanShot 2023-10-19 at 21 40 52@2x](https://github.com/nextcloud/server/assets/48948959/e1bfd513-c940-47c8-8870-8712e00c10d4)

**Comment by JuliaKirschenheuter** [comment]
Closing this issue because this is already implemeted. Opened another issue regarding keyboard support: 

- https://github.com/nextcloud/server/issues/41012
